nodeup/pkg/model/context.go

```diff
@@ -85,3 +91,83 @@ func (c *NodeupModelContext) NetworkPluginDir() string {
 		return "/opt/cni/bin/"
 	}
 }
+
+func (c *NodeupModelContext) BuildPKIKubeconfig(id string) (string, error) {
```

Comment:
It's coming - it's on my integration branch where I comingled RBAC and etcd3.

I'm actually kinda surprised it hasn't been in any of the PRs yet!  Soon!